source/computer/fedora-setup.rst

```diff
@@ -192,6 +192,24 @@ Fedora 的归档管理器可以识别并打开 Linux 下的常见压缩格式（
 
     $ sudo dnf install unar
 
+终端
+^^^^^
+
+Fedora 自带的终端模拟器是 GNOME Terminal，使用起来中规中矩。
+日常科研经常需要开好几个终端，切换和管理起来比较麻烦。
+
+`Terminator <https://github.com/gnome-terminator/terminator>`__
```

Comment:
```suggestion
`Terminator <https://gnome-terminator.org/>`__
```

Reply:
Also need a link of the documentation page (https://gnome-terminator.readthedocs.io/en/latest/).

Reply:
https://gnome-terminator.org/ is an older site. The readers also can find it on Github (right side)

Reply:
No. It's the latest site.